lib/assets/csv_script.rb

```diff
@@ -54,8 +54,25 @@
 content = csv_lines[1..-1]
 content.shuffle!
 
+def get_chunks(content)
+  # convert string with square brackets to int array
+  increments = CHUNK_INCREMENTS.gsub(/[\[\] ]/, "").split(",").map { |x| x.to_i }
+  current_index = 0
+  chunks = []
+  increments.each do |increment|
+    if current_index + increment <= content.length
+      chunks << content[current_index, increment]
+      current_index += increment
+    end
+  end
+  if current_index < content.length
+    chunks.push content[current_index..-1]
```

Comment:
Minor: This can also use the `<<` operator like line 64, right?

Reply:
There were some small things I spotted to clean up so I included that too